### ISSUE TITLE
development halted

### ISSUE THREAD

**Issue by lateleite** [Ochii/cso2-master-server#145]
as you may have noticed, i've stopped working on anything CSO2 related last year or two, and i don't expect to resume anytime soon.
feel free to fork this master server, though i find it way too complicated due to it's architecture and its host language (the launcher is also complicated, but to a lesser extent).
i'm also shutting down cso2.leite.xyz since it was only used by me as testing grounds.

moving on, i only know about KouKouChan's master server (it disappeared for whatever reason, found it archived at https://github.com/MouseFork/CSO2-Server ), so feel free post any information you have on any other CSO2 related projects.

thanks for reading

**Comment by lateleite** [comment]
someone requested a packet dump i did a couple weeks before the chinese CSO2 server closed to be made public: [sanitized_packets.tar.gz](https://github.com/L-Leite/cso2-master-server/files/8969845/sanitized_packets.tar.gz)
these are dumps of packets received between logging in and playing a (bot) practice match

**Comment by IGNRexI** [comment]
ohh..

**Comment by IGNRexI** [comment]
Is there a place where we can contact you?
The reason the KouKouChan server was shut down: "The hackers were selling the project for money without the developer's knowledge" and I'm still trying to develop the server myself. I fully integrated the relay system you coded before. I solved the relay problems in the c++ launcher. I can't code for myself.
but I'm still trying to learn and make the features.
my discord: https://cso2.net/discord

**Comment by LoveBeforT** [comment]
> Is there a place where we can contact you? The reason the KouKouChan server was shut down: "The hackers were selling the project for money without the developer's knowledge" and I'm still trying to develop the server myself. I fully integrated the relay system you coded before. I solved the relay problems in the c++ launcher. I can't code for myself. but I'm still trying to learn and make the features. my discord: https://discord.gg/csonline2

you can use Source SDK‘s leak code to write a Dedicated Server, there is a corresponding interface in engine.dll.

**Comment by IGNRexI** [comment]
can we chat? Can you come to discord? @LoveBeforT

**Comment by IGNRexI** [comment]
if you stopped cso2 server development at least give me the latest version @LoveBeforT 
I am using v.5.0 old version. 'Can you give me the v.8.0 source code via discord or mail?'
http://discord.gg/csonline2
WASSUP#3596
kopgelaslan@gmail.com

**Comment by IGNRexI** [comment]
> > Is there a place where we can contact you? The reason the KouKouChan server was shut down: "The hackers were selling the project for money without the developer's knowledge" and I'm still trying to develop the server myself. I fully integrated the relay system you coded before. I solved the relay problems in the c++ launcher. I can't code for myself. but I'm still trying to learn and make the features. my discord: https://discord.gg/csonline2
> 
> you can use Source SDK‘s leak code to write a Dedicated Server, there is a corresponding interface in engine.dll.

What do you mean by corresponding interface? Can you summarize?
What does the corresponding interface mean?
